runtime/karura/src/lib.rs

```diff
@@ -1495,7 +1495,7 @@ parameter_types! {
 	pub const LKSMCurrencyId: CurrencyId = CurrencyId::Token(TokenSymbol::LKSM);
 	pub MinimumMintThreshold: Balance = 10 * cent(KSM);
 	pub RelaychainSovereignSubAccount: MultiLocation = create_x2_parachain_multilocation(RelaychainSubAccountId::HomaLite as u16);
-	pub MaxRewardPerEra: Permill = Permill::from_rational(500u32, 1_000_000u32); // 1.2 ^ (1/365) = 1.0004996359
+	pub MaxRewardPerEra: Permill = Permill::from_rational(500u32, 1_000_000u32); // 1.2 * (1/365) = 1.0004996359
```

Comment:
<img width="818" alt="Screen Shot 2021-08-24 at 3 06 21 PM" src="https://user-images.githubusercontent.com/1201310/130549380-39c1a47b-8bd9-4fde-8c7a-3753c86aa4c4.png">

Reply:
Yeah the original is correct. i.e. 1.0005 ^ 365 = approx (1 + 20%)